GAE/src/locale/pt.properties

```diff
@@ -99,48 +96,51 @@ Choose\ cascade\ resource = Escolher o recurso cascata
 Choose\ data\ approval\ group = Escolher grupo de aprova��o
 Choose\ test\ type = Escolha o tipo de teste
 Choose\ the\ registration\ form = Selecionar formul�rio de registo
+Clean\ your\ data\ using\ the\ Data\ cleaning\ export.\ Here\ you\ can\ import\ the\ cleaned\ data\ back\ to\ your\ Flow\ form.\ Keep\ the\ structure\ of\ your\ Data\ cleaning\ export.\ The\ file\ must\ be\ .xlsx\ or\ .xls\ format.\ The\ import\ will\ override\ the\ existing\ data. = Limpe seus dados usando a Limpeza de dados. Aqui voc� pode importar os dados limpos de volta para o formul�rio Flow. Mantenha a estrutura da sua exporta��o de limpeza de dados. O arquivo deve estar no formato .xlsx ou .xls. A importa��o substituir� os dados existentes.
 Clear\ All = Limpar tudo
+Click\ "New\ export"\ to\ get\ started = Clique em "Nova exporta��o" para come�ar
 Close = Fechar
 Close\ notifications = Fechar notifica��es 
 Close\ window = Fechar janela
 Close\ without\ saving = Fechar sem gravar
 Code = C�digo
-Codes\ are\ missing\ for\ these\ options = C�digos em falta para essas op��es
 Codes\ can\ only\ contain\ letters\ A\ -\ Z,\ a\ -\ z\ and\ digits\ 0\ -\ 9 = Os c�digos apenas podem conter letras de A-Z, a-z e d�gitos de 0-9
 Collapse = Ocultar
 Collected = Colectado
 Collected\ from = Colectadas de
 Collected\ on = Colectado a
-Collection\ period = Periodo de colecta
 Combines\ options\ and\ cascade\ levels\ into\ a\ single\ column.\ Puts\ repeated\ groups\ into\ separate\ sheets. = Combina op��es e n�veis de cascata em uma �nica coluna. Coloca grupos repetidos em folhas separadas.
 Comprehensive\ report = Relat�rio abrangente
+Connect\ the\ device\ to\ the\ PC.\ In\ the\ 'Android'\ folder\ go\ to\ 'data',\ to\ 'org.akvo.flow'\ and\ in\ 'files'.\ Copy\ over\ the\ 'published'\ folder\ to\ the\ PC.\ Then\ create\ a\ zip\ file\ from\ that\ folder. = Conecte o dispositivo ao PC. Na pasta 'Android', v� para 'dados', 'org.akvo.flow' e em 'arquivos'. Copie a pasta 'published' para o PC. Em seguida, crie um arquivo zip a partir dessa pasta.
```

Comment:
I think 'dados' should be still 'data' and 'arquivos' 'files' because we do not translate these names in the app